sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/parser/ParserUtils.scala

```diff
@@ -43,6 +44,14 @@ object ParserUtils {
     new ParseException(s"Operation not allowed: $message", ctx)
   }
 
+  /** Check if duplicate keys exist in a set of key-value pairs. */
+  def checkDuplicateKeys(
+      keyPairs: mutable.Buffer[_ <: (String, Any)], ctx: ParserRuleContext): Unit = {
```

Comment:
Style: Parameter per line.

`mutable.Buffer[_ <: (String, Any)]` is it possible to just check a `Seq` or `Traversable`?  Do you think it is possible to improve the signature (maybe move the type of Any into a generic type)? It is kinda messy now.